volumina/pixelpipeline/datasources.py

```diff
@@ -353,7 +353,7 @@ def put( self, slicing, array ):
             inputKeys = [tag.key for tag in inputTags]
             transposedArray = taggedArray.withAxes(*inputKeys)
     
-            taggedSlicing = dict(zip('txyzc', slicing))
+            taggedSlicing = dict(list(zip('txyzc', slicing)))
```

Comment:
This is an automatic change by python utility - 2to3
In principle I agree with you